gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,21 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm' '')"
+	backends+="$(usex libinput ',libinput' '')"
+	backends+="$(usex x11-backend ',x11' '')"
+	# Get rid of leading comma
+	backends="$(echo ${backends} | sed -e 's/^,//')"
 	# xcb-util-errors is not on Gentoo Repository (and upstream seems inactive?)
 	local emesonargs=(
 		"-Dxcb-errors=disabled"
```

Comment:
this is my error, I didn't remove the static disable of xcb-errors when I have the usex on the next line, but the error still exists here.

Reply:
Will remove in the next commit

Reply:
done